src/hooks/useActivitiesLocalStorage.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import App from '../App'
+import { createMemoryHistory } from 'history'
+import { Router } from 'react-router'
+
+describe('App.js', () => {
+  beforeEach(() => {
+    localStorage.clear()
+  })
+
+  it('sets activities to local storage', () => {
+    const history = createMemoryHistory()
```

Comment:
Vielelicht kannst du die Zeile hier auch mit in before each packen!

Reply:
Gute Idee. Probiere ich auf jeden Fall.